src/schema/v2/order/submitOrderMutation.ts

```diff
@@ -0,0 +1,49 @@
+import { GraphQLString, GraphQLNonNull, GraphQLID } from "graphql"
+import {
+  ORDER_MUTATION_FLAGS,
+  OrderMutationResponseType,
+} from "./sharedOrderTypes"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+
+interface Input {
+  id: string
+  confirmationToken?: string
+}
+
+export const submitOrderMutation = mutationWithClientMutationId<
+  Input,
+  any,
+  ResolverContext
+>({
+  name: "submitOrder",
+  description: "Submit an order",
+  inputFields: {
+    id: { type: new GraphQLNonNull(GraphQLID), description: "Order id." },
+    confirmationToken: {
+      type: GraphQLString,
+      description: "Stripe confirmation token.",
+    },
+  },
+  outputFields: {
+    orderOrError: {
+      type: OrderMutationResponseType,
+      resolve: (response) => response,
+    },
+  },
+  mutateAndGetPayload: async (input, { meOrderSubmitLoader }) => {
```

Comment:
We should be using our canonical `ThingOrError` patterns for mutations (see here: https://github.com/artsy/metaphysics/pull/6564)

Reply:
I think it is using the `orderOrError` pattern (see tests below), but happy to update if I missed something!